src/Sylius/Behat/Service/Accessor/EmailChecker.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service\Accessor;
+
+use Behat\Mink\Session;
+use Symfony\Component\HttpKernel\Client;
+use Symfony\Component\HttpKernel\Profiler\Profiler;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class EmailChecker implements EmailCheckerInterface
+{
+    /**
+     * @var Profiler
+     */
+    private $symfonyProfiler;
+
+    /**
+     * @var Session
+     */
+    private $session;
+
+    /**
+     * @param Profiler $symfonyProfiler
+     * @param Session $session
+     */
+    public function __construct(Profiler $symfonyProfiler, Session $session)
+    {
+        $this->symfonyProfiler = $symfonyProfiler;
+        $this->session = $session;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasRecipient($recipient)
+    {
+        $messages = $this->getMessages();
+        foreach ($messages as $message) {
+            if (array_key_exists($recipient, $message->getTo())) {
```

Comment:
``$message->getTo()`` has the following output ``['email@exampl.com' => null]`` 😞 .